translate/cloud-client/beta_snippets.py

```diff
@@ -135,7 +135,7 @@ def list_languages_with_target(project_id, display_language_code):
     print('Supported Languages:')
     for language in response.languages:
         print('Language Code: {}'.format(language.language_code))
-        print('Display Name: {}\n'.format(language.display_name))
+        # print('Display Name: {}\n'.format(language.display_name))
```

Comment:
Should this be commented out?

Reply:
It was intentional. I was unable to find a way to work around non-ASCII character printing errors. This is meant as a temporary fix.

Reply:
[resolving bc only relevant to python2; I've reverted in the most recent commit]